src/types/search.types.ts

```diff
@@ -4,15 +4,17 @@ interface SearchResultContributor {
 }
 
 export interface SearchResult {
+  id: string;
+  contributor: SearchResultContributor[];
+  date: string;
+  owner: string;
+  title: string;
+}
+
```

Comment:
jeg syns det er rart at `SearchResult` og `LatestPublication` ikke er en og samme type... begge er en forenklet representasjon av en publikasjon, men det er mer en task for backend evt

Reply:
de er helt forskjellige. Derfor jeg splittet opp i to forskjellige typer. Backend sin jobb å eventuelt fikse dette